2024-10-28 01:30:16,676 - system - INFO - DEBUGGING THIS APP
2024-10-28 01:30:16,676 - system - INFO - Python executable: /usr/local/bin/python
2024-10-28 01:30:16,676 - system - INFO - Python version: 3.9.20 (main, Oct 19 2024, 01:00:38) 
[GCC 12.2.0]
2024-10-28 01:30:16,676 - system - INFO - Virtual environment: None
2024-10-28 01:30:16,676 - system - INFO - Testing this - localhost
2024-10-28 01:30:16,711 - system - INFO - Received request
2024-10-28 01:30:16,711 - system - INFO - Request method: POST
2024-10-28 01:30:16,712 - system - INFO - Request headers: Host: process-daily-narrative-418435618601.us-central1.run.app
Content-Type: application/json
X-Cloudscheduler: true
X-Cloudscheduler-Scheduletime: 2024-10-27T18:30:00-07:00
X-Cloudscheduler-Jobname: hourly_qualia_update
Content-Length: 53
User-Agent: Google-Cloud-Scheduler
X-Cloud-Trace-Context: 4fad91e12d3cfae3b25c32a1dd7401d3/6651539679150878715;o=1
Traceparent: 00-4fad91e12d3cfae3b25c32a1dd7401d3-5c4f0418c9342ffb-01
X-Forwarded-For: 35.243.23.205
X-Forwarded-Proto: https
Forwarded: for="35.243.23.205";proto=https
Accept-Encoding: gzip, deflate, br


2024-10-28 01:30:16,712 - system - INFO - Request data: b'{\n  "REQUEST_TYPE": "QUALIA_UPDATE",\n  "PARAMS": {}\n}'
2024-10-28 01:30:16,712 - system - INFO - Request JSON: {'REQUEST_TYPE': 'QUALIA_UPDATE', 'PARAMS': {}}
2024-10-28 01:30:16,712 - system - INFO - REQUEST_TYPE: QUALIA_UPDATE
2024-10-28 01:30:16,712 - system - INFO - PARAMS: {}
2024-10-28 01:30:16,712 - system - INFO - Applying function: hourly_update
2024-10-28 01:30:16,712 - system - INFO - Updating hourly...
2024-10-28 01:30:16,868 - system - INFO - [[<Aggregation alias=unprocessed_count, value=0.0, readtime=2024-10-28 01:30:16.854919+00:00>]]
2024-10-28 01:30:16,868 - system - INFO - Processing 0 out of 0.0 events for retrieved_reddit_posts
2024-10-28 01:30:16,998 - system - INFO - [[<Aggregation alias=unprocessed_count, value=0.0, readtime=2024-10-28 01:30:16.985243+00:00>]]
2024-10-28 01:30:16,998 - system - INFO - Processing 0 out of 0.0 events for retrieved_articles
2024-10-28 01:30:17,106 - system - ERROR - Error in QUALIA_UPDATE {}
2024-10-28 01:30:17,116 - system - ERROR - Traceback (most recent call last):
  File "/app/main.py", line 51, in call_default_cloud_run
    ret_val = func_to_apply(**request_params)
  File "/app/src/scripts.py", line 18, in hourly_update
    update_qualia()
  File "/app/utils/qualia.py", line 304, in update_qualia
    events_to_process = get_events_to_process()
  File "/app/utils/qualia.py", line 170, in get_events_to_process
    event_dict = doc.to_dict()
UnboundLocalError: local variable 'doc' referenced before assignment

